src/fragments/lib/graphqlapi/flutter/authz/20_oidc.mdx

```diff
@@ -1,7 +1,16 @@
-<Callout>
+To start, create a provider class inheriting from `OIDCAuthProvider`.
 
-OIDC mode is not yet supported for Flutter. We are actively working on this. 
+```dart
+import 'package:amplify_api/amplify_api.dart';
 
-We have created a [Github Issue](https://github.com/aws-amplify/amplify-flutter/issues/308) to track this missing feature. 
+class CustomOIDCProvider extends OIDCAuthProvider {
+  const CustomOIDCProvider();
 
-</Callout>
\ No newline at end of file
+  @override
+  Future<String?> getLatestAuthToken() async => '[OPEN-ID-CONNECT-TOKEN]';
```

Comment:
Should we explain briefly how to get this token?  In case a developer doesn't have context knowledge here

Reply:
For sure- right below this there is an example using Cognito 👍